components/brave_wallet_ui/stories/mock-data/mock-networks.ts

```diff
@@ -0,0 +1,18 @@
+import { EthereumChain } from '../../constants/types'
+import {
+  ETHIconUrl
+} from '../../assets/asset-icons'
+export const mockNetworks: EthereumChain[] = [
```

Comment:
now we can use selectedNetwork or networkList with real data instead of mocks

Reply:
Storybook needs this mockData to work! 😊